tests/main.test.ts

```diff
@@ -2,11 +2,11 @@ import { expect, test } from '@playwright/test';
 
 test('SvelteFire app initializes properly', async ({ page }) => {
 	await page.goto('/');
-	expect(await page.textContent('h1')).toBe('Welcome to SvelteFire');
+	await expect(page.locator('h1')).toHaveText('Welcome to SvelteFire');
```

Comment:
This got changed from a manual expect to a Playwright web-first assertions expect.